Slim/Plugin/CLI/Plugin.pm

```diff
@@ -278,63 +281,62 @@ sub cli_socket_accept {
 	}
 }
 
+sub client_socket_keepalive {
+	my $client_socket = shift;
+
```

Comment:
Please add a `killTimers()` call here, too. We've had problems in the past where insane numbers of timers got created for whatever reason. Better be safe than sorry.

Reply:
I can also not do the timer think at all and see how it goes. I've tested that and when the KEEPALIVE kicks-in, the socket is closed (at least on Windows). So we can see what happens.